assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -178,23 +88,11 @@ function DropDown({
   lightMode = false,
   placeholder,
 }) {
-  DropDown.wrapperRef = useRef(null);
```

Comment:
Not directly related to this PR, but such cases need to be found and changed to `const x = useRef()`. This is potentially pretty harmful.